src/main/java/ReviewResponse.java

```diff
@@ -13,6 +16,8 @@ public class ReviewResponse {
     @Getter
     String okbb;
 
+    @Setter
+    String blabla;
```

Comment:
Directly exposing class fields like 'blabla' without methods for access control violates Encapsulation principle of SOLID. (Severity: medium)